### DISCUSSION TITLE
Delegated Balance Rounding Bug

### DISCUSSION THREAD

**Discussion by arcturiano** [BitCannaGlobal/WeedWallet#86] [General]
I have delegated exactly 6000 BCNA as can be seen in by delegation historical two transactions, but My Delegation says 5999.999999 BCNA

I think this is a rounding issue, don't think it has to do with my validator or that it is due to gas 

![DelegatedBalanceRounding](https://github.com/BitCannaGlobal/WeedWallet/assets/94879750/0db7de84-a853-4fbe-9d46-b471e0571db1)

**Comment by TJ-BitCanna** [comment]
We have been struggling with this earlier already, but we'll dive into it to check to be sure again.

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- recheck the rounding bug (how do Keplr & CosmoStation solve this?)

**Comment by TJ-BitCanna** [comment]
This is not a problem caused by the WeedWallet, but seems to be caused in the blockchain itself.

We will check with the SDK-team to investigate.

@RaulBernal can you check with the SDK-team?
